laravel-live-coding/tests/Unit/UserControllerTest.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+namespace Tests\Unit;
+
+use App\Repositories\UserRepository;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+class UserControllerTest extends TestCase
+{
+    use RefreshDatabase;
+
+    public function testIndexReturnsCorrectResponse()
+    {
+        // Arrange
+        $mockUserData = [
+            [
+                'name' => 'Dima',
+                'email' => '@gmail.com',
+                'phone' => '+38000000000',
+                'photo' => 'https://d3jqtupnzefbtn.cloudfront.net/andersenlab/new-andersensite/testimonials/60x60/without-photo.png',
+                'wallets' => [
+                    [
+                        'currency' => 'USD',
+                        'amount' => 12321314124134e312321,
+                    ],
+                    [
+                        'currency' => 'EUR',
+                        'amount' => 12321314124134e312321,
+                    ],
+                ],
+            ],
+            [
+                'name' => 'Mariya',
+                'email' => '@gmail.com',
+                'phone' => '+3800312300000',
+                'photo' => 'https://d3jqtupnzefbtn.cloudfront.net/andersenlab/new-andersensite/testimonials/60x60/without-photo.png',
+                'wallets' => [
+                    [
+                        'currency' => 'USD',
+                        'amount' => 12321314124134e312321,
+                    ],
+                    [
+                        'currency' => 'EUR',
+                        'amount' => 12321314124134e312321,
+                    ],
+                ],
+            ],
+        ];
+
```

Comment:
instead of this, try connecting to sqlLite(it will be your test DB) and push some Mock data, but use different tools to push information there